extension/src/tree/index.ts

```diff
@@ -61,6 +61,9 @@ export const getCliErrorTreeItem = (
     command: RegisteredCommands.EXTENSION_SHOW_OUTPUT,
     title: 'Show DVC Output'
   }
+
+  treeItem.label = label
```

Comment:
Any way to test for this issue? I couldn't come up with anything

Reply:
Oh, looks like my original review never got sent in 🤣. As mentioned in https://github.com/iterative/vscode-dvc/pull/4522#discussion_r1296543485 going to look into testing this with `getTreeItem`.